spec/Vehicle/DigitalKey.vspec

```diff
@@ -66,7 +79,7 @@ DigitalKey.VehicleId:
   type: attribute
   description: Identifier used to identify the vehicle for digital key purposes.
   comment: The vehicle identifier used for key purposes is not necessarily the same as any identifier used
-           for other purposes. It is possibly not the same as VIN.
+           for other purposes. For privacy and security reasons, it is likely not the same as the VIN.
```

Comment:
I know that VIN in some countries is considered sensitive from a privacy perspective. But is there really a security reason to not use VIN?